File: API/ios-tms.Core/Domain/Address.cs
using System;
namespace iOS_TMS.Core.Domain;

public class Address : EntityBase
{
    public string AddressLineOne { get; set; }
    public string AddressLineTwo { get; set; }
    public string City { get; set; }
    public string State { get; set; }
    public string ZipCode { get; set; }
}

File: App/ios-tms/Models/Address.swift
//
//  Address.swift
//  ios-tms
//
//  Created by Kyle Morton on 8/28/22.
//

import Foundation

struct Address: Codable, Identifiable {
    var id: Int
    var addressLineOne: String
    var addressLineTwo: String
    var city: String
    var state: String
    var zipCode: String
    
    var label: String {
        return "\(city), \(state) \(zipCode)"
    }
    
    init(id: Int, city: String, state: String, zipCode: String) {
        self.id = id
        self.city = city
        self.state = state
        self.zipCode = zipCode
        self.addressLineOne = ""
        self.addressLineTwo = ""
    }
}
